clients/cobol-lsp-vscode-extension/src/__tests__/services/copybook/CopybookDownloadServiceTest.spec.ts

```diff
@@ -325,7 +329,7 @@ describe("Test the creation of folders that contains copybooks downloaded from M
   function setupScenario() {
     const testFolder = path.join(
       __dirname,
```

Comment:
Is there a temporary directory that could be used here, so the tests do not modify the source tree??

Reply:
I agree but should we do this in scope of this PR ?